fix: reduce website footer

deboer-tim opened the pull request:
### What does this PR do?

Ok, here's my pass at reducing the vertical height of the footer. Long term I think we can and should do more, this is just a quick initial cleanup pass:
- Left: removed three links (so both columns are down to 5). These are a judgement call, e.g. I felt like 'Docker migration' is a bit of marketing we'd want to leave for now.
- Right: remove Blog (it's already at the top of the page).
- Right: remove planning and roadmap, both not really useful link and accessible from the Github link above.
- Put copyright and license on one line.

Overall, this reduces the footer height from 11 lines of text to 7.

### Screenshot/screencast of this PR

Before:

<img width="1257" alt="Screenshot 2023-10-27 at 4 30 37 PM" src="https://github.com/containers/podman-desktop/assets/19958075/02efe459-6ead-4bfc-986a-27916a3a3abf">

After (updated to latest commit):

<img width="1056" alt="Screenshot 2023-10-30 at 2 43 53 PM" src="https://github.com/containers/podman-desktop/assets/19958075/3249576a-3162-4de2-a427-2cf9d56536c7">

### What issues does this PR fix or reference?

Fixes #4299.

### How to test this PR?

`yarn website:dev`

benoitf reviewed (commented):
IMHO I think roadmap is interesting as a direct link

but probably for Chat, we should reduce the numbers of links as it looks like it's too many

jeffmaury commented:
> IMHO I think roadmap is interesting as a direct link
> 
> but probably for Chat, we should reduce the numbers of links as it looks like it's too many

Keep the discord one and replace the others chat links by a link to https://github.com/containers/podman-desktop#communication

deboer-tim commented:
@benoitf I didn't think the ([previous link](https://github.com/containers/podman-desktop/projects?type=beta)) was very useful. How about pointing them to the [planning board](https://github.com/orgs/containers/projects/4) or [current sprint](https://github.com/orgs/containers/projects/4/views/8) instead?

@jeffmaury Makes sense, will do.

benoitf commented:
@deboer-tim yes as you suggested, we need a redirect link, current link is not good 👍  (I would prefer 'current sprint' link as default to backlog is not really great as well (you need to click on a link again)

I agree that footer links are like direct calls and as Jeff pointed out, when there are several links, just take one

deboer-tim commented:
Pushed another commit:
- Reduced four chat links to just Discord and a link to Communication section in the readme. Added back in a link to the sprint planning board.
- Removed Compose from the doc links to even things out again.

slemeur commented:
Good simplification!
